app/apiService.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
         const response = await _getRequest('/user/search?includeInactive=true&username=' + username)
         .catch(error => {
             const response = error.response;
```

Comment:
I don't think it would ever happen that the response is null or undefined, but do you think we should be preemptive and change this to something like the following?
```suggestion
            const response = error?.response;
```

Reply:
You mean _I don't think it would ever happen that the `error` is null or undefined_? Not the `response`, right? Cause `response` might be `undefined` actually.

Anyway, if it's regarding `error` I agree with the change 😄

Reply:
oops, yeah, that's what I meant to say 😅

Reply:
Nice 👍 applied in 4e41c2e . Thanks 🙂